openedx/core/djangoapps/enrollments/api.py

```diff
@@ -527,6 +535,23 @@ def is_enrollment_valid_for_proctoring(username, course_id):
     if enrollment['mode'] not in appropriate_modes:
         return False
 
+    # Only continue if a proctored exam is found in the course
+    has_proctored_exam = False
+
+    timed_exams = modulestore().get_items(
+        course_id,
+        qualifiers={'category': 'sequential'},
+        settings={'is_time_limited': True}
+    )
+
+    for exam in timed_exams:
```

Comment:
You can simplify this a bit by writing something like,

`has_proctored_exam = any([exam.is_proctored_exam for exam in timed_exams])`.

Here are the [docs](https://docs.python.org/3/library/functions.html#any) for the built in `any` function.